spec/paperclip/has_attached_file_spec.rb

```diff
@@ -34,31 +34,41 @@
       assert_adding_attachment('avatar').defines_callback('before_destroy')
     end
 
-    it 'defines an after_commit callback' do
+    it 'defines the Paperclip-specific callbacks' do
+      assert_adding_attachment('avatar').defines_callback('define_paperclip_callbacks')
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.